cmd/warpforge/status.go

```diff
@@ -10,7 +10,9 @@ import (
 	"github.com/fatih/color"
 	"github.com/urfave/cli/v2"
 	"github.com/warpfork/warpforge/pkg/formulaexec"
+	"github.com/warpfork/warpforge/pkg/tracing"
 	"github.com/warpfork/warpforge/wfapi"
+	"go.opentelemetry.io/otel/trace"
```

Comment:
Nit: I'm beginning to gain a desire that we enforce goimports in CI.  (e.g. imports from this same module go in a separate block.)

Reply:
Which `goimports` version/invocation are you using? Mine does not do this. I think it respects line separation if you do it manually but it won't enforce it.

Reply:
:see_no_evil: I'm not using it either; I thought that was a feature, though.  Maybe I don't mean `goimports` and I'm thinking of some other tool...